src/Client/ClientBase.cpp

```diff
@@ -3000,7 +3041,7 @@ void ClientBase::init(int argc, char ** argv)
 
         ("query,q", po::value<std::vector<std::string>>()->multitoken(), R"(query; can be specified multiple times (--query "SELECT 1" --query "SELECT 2"...))")
         ("queries-file", po::value<std::vector<std::string>>()->multitoken(), "file path with queries to execute; multiple files can be specified (--queries-file file1 file2...)")
-        ("multiquery,n", "If specified, multiple queries separated by semicolons can be listed after --query. For convenience, it is also possible to omit --query and pass the queries directly after --multiquery.")
+        ("multiquery,n", "Obsolete (does nothing, multiquery is the default)")
```

Comment:
"multiquery is the default" is slightly confusing here.

Please
- replace it by "Obsolete, does nothing" and
- in l. 3042, please use this description: "query; can be specified multiple times (--query "SELECT 1" --query "SELECT 2"); it is also possible to pass multiple queries separated by semicolon (--query "SELECT 1; SELECT 2;")`